phabricator/__init__.py

```diff
@@ -228,6 +229,16 @@ def __init__(self, api, interface=None, endpoint=None, method=None, nested=False
         self.endpoint = endpoint
         self.method = method
         self.nested = nested
+        self.session = requests.Session()
+        retry_strategy = Retry(
```

Comment:
Could move this inline to the `HTTPAdapter` constructor since it's not used anywhere else and it doesn't make it too unreadable.

Reply:
done :)